lib/features/profiles/widgets/profile_box.dart

```diff
@@ -21,12 +21,18 @@ class ProfileBox extends StatelessWidget {
             crossAxisAlignment: CrossAxisAlignment.stretch,
             children: [
               SizedBox(height: 8),
-              Text(
-                title,
-                style: Theme.of(context).textTheme.titleMedium,
+              Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 16),
+                child: Text(
+                  title,
+                  style: Theme.of(context).textTheme.titleMedium,
+                ),
               ),
               SizedBox(height: 8),
```

Comment:
:upside_down_face: you could remove the  sized boxes and use `vertical: 8` padding